oop/src/views/MoveInfoPage.vue

```diff
@@ -247,6 +241,14 @@ export default {
       
       setTimeout(() => {
         this.isLoading = false;
+
+        // Eliminate empty address lines.
+        const currNumOfAddressLines = Math.max(MIN_ADDRESS_LINES, this.addressLines.length);
+        for (let i=currNumOfAddressLines-1; i>=0; i--){
+          if ((this.addressLines[i].value == null || this.addressLines[i].value == '') && currNumOfAddressLines > 1){
+            this.addressLines.splice(i,1);
+          }
+        }
```

Comment:
I've made some changes for this file as well because of the address line bug that Neha found. Was this code put here to fix that bug?